decidim-proposals/lib/decidim/proposals/proposal_answer_creator.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # This class is responsible for creating the imported proposal answers
+    # and must be included in proposals component's import manifest.
+    class ProposalAnswerCreator < Decidim::Admin::Import::Creator
+      def initialize(data, context = nil)
```

Comment:
Maybe this could be the default initialize method at `Decidim::Admin::Import::Creator` and then we can customize it for `Decidim::Proposals::ProposalCreator`?

Reply:
Yes